packages/paste-website/src/components/color-swatch/ColorGradient.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import {styled, themeGet} from '@twilio-paste/styling-library';
+import {colors} from '@twilio-paste/design-tokens';
+import {colors as darkColors} from '@twilio-paste/design-tokens/dist/themes/dark/tokens.es6';
+import {Box} from '@twilio-paste/box';
+import {useDarkModeContext} from '../../context/DarkModeContext';
+
+const aliasPrefixes = ['colorGray', 'colorRed', 'colorOrange', 'colorYellow', 'colorGreen', 'colorBlue', 'colorPurple'];
+
+type AliasName = keyof typeof colors;
+
+const sortAliasNames = (aliasNames: string[]): string[] => {
+  return aliasNames.sort((nameA, nameB) => nameA.localeCompare(nameB, undefined, {numeric: true})).reverse();
+};
+
+const getAliasValuesFromPrefix = (prefix: string, theme: 'default' | 'dark'): string[] => {
+  const themeColors = theme === 'default' ? colors : darkColors;
```

Comment:
Tried to future proof it so if the dark palette colors change the gradient will look correct.